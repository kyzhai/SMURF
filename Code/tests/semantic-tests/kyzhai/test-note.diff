1c1,10
< Fatal error: exception Sast.Main_missing
---
> Program: 
> 	SVardef: 
> 		ID: a :: Note
> 
> 	 (1, 2)$4...
> Global Scope: 
> 	ID: a :: Note
> 	ID: print :: Unknown
> 	ID: random :: Unknown
> 
