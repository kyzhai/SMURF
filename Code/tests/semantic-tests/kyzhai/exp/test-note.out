Program: 
	SVardef: 
		ID: a :: Note

	 (1, 2)$4...
Global Scope: 
	ID: a :: Note
	ID: print :: Unknown
	ID: random :: Unknown

